include/battle.h

```diff
@@ -754,9 +754,9 @@ struct BattleStruct
     u8 timesGotHit[NUM_BATTLE_SIDES][PARTY_SIZE];
     u8 stickySyrupdBy[MAX_BATTLERS_COUNT];
     u8 supremeOverlordCounter[MAX_BATTLERS_COUNT];
-    u8 shellSideArmCategory[MAX_BATTLERS_COUNT][MAX_BATTLERS_COUNT];
+    enum DamageCategory shellSideArmCategory[MAX_BATTLERS_COUNT][MAX_BATTLERS_COUNT];
```

Comment:
Since it's impossible to specify the amount of bits for array elements, this suddenly takes up a lot more space. We can revert it to u8 and just add a comment explaining why, perhaps?

Reply:
Yeah, that's reasonable :)

If we decided that we wanted the space and the type checking we could instead spell it:
```c
struct __attribute__((packed)) { enum DamageCategory category:8} shellSideArm[MAX_BATTLERS_COUNT][MAX_BATTLERS_COUNT];
```
And then `shellSideArmCategory[i][j]` becomes `shellSideArm[i][j].category`.